js/bigone.js

```diff
@@ -266,6 +309,100 @@ module.exports = class bigone extends Exchange {
             };
             result.push (entry);
         }
+        const contracts = promises[1];
+        //
+        //     [
+        //        {
+        //             "nextFundingRate": 0.00043725,
+        //             "usdtPrice": 0.999742233120400266,
+        //             "turnover24h": 1965.314165971383,
+        //             "openInterest": 1186214.0,
+        //             "nextFundingTime": 1658023200000,
+        //             "volume24hInUsd": 41618417,
+        //             "fundingRate": 0.00047145,
+        //             "volume24h": 41212785,
+        //             "ethPrice": 1345.31,
+        //             "last24hMaxPrice": 21578,
+        //             "btcPrice": 21176.47,
+        //             "latestPrice": 21208.0,
+        //             "symbol": "BTCUSD",
+        //             "last24hPriceChange": 0.0146154766176294701,
+        //             "openValue": 36.0977933512444842,
+        //             "last24hMinPrice": 20510.5,
+        //             "markPrice": 21180.90710576,
+        //             "indexPrice": 21176.47
+        //         },
+        //         ...
+        //     ]
+        //
+        for (let i = 0; i < contracts.length; i++) {
+            // Assumes that contract is settled in USDT if USDT is present in symbol
```

Comment:
This is not my favorite, but it's the best way I thought to parse the market

Reply:
@pcriadoperez can we at least add it as an option, like 'defaultSettle'?